tests/javascript_tests/test_color_view_controller.js

```diff
@@ -21,7 +21,7 @@ $(document).ready(function() {
                   ['PC.635', 'YATGCTGCCTCCCGTAGGAGT', 'Fast', '20071112']];
       decomp = new DecompositionModel(name, ids, coords, pct_var, md_headers,
                                       metadata);
-      var dv = new DecompositionView(decomp);
+      dv = new DecompositionView(decomp);
```

Comment:
It has to do to what you mentioned with your PR about globals. In this case, we need dv as global for the tests and that var makes it local.